lib/mod.ts

```diff
@@ -22,6 +22,7 @@ export interface MergePullRequestOptions {
   owner: string;
   repo: string;
   number: number;
+  mergeMethod: string;
```

Comment:
Maybe we could add a `MergeMethod` enum just to make it simpler to know the options available?

Reply:
That sounds like a sound approach

Reply:
fixed